package.json

```diff
@@ -50,12 +51,12 @@
   },
   "devDependencies": {
     "@types/axios": "^0.14.0",
+    "@types/dompurify": "^2.3.0",
```

Comment:
same.

Reply:
it was just moved by prettier I didn't change it